### PR TITLE
chore: Build status

### PR DESCRIPTION
**Pull request by dj-stormtrooper**
#### Summary

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Test locally on your own infrastructure
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation)) 
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 The open-source cloud asset inventory powered by SQL.
 
-![BuildStatus](https://img.shields.io/github/workflow/status/cloudquery/cloudquery/test?style=flat-square)
+![BuildStatus](https://img.shields.io/github/workflow/status/cloudquery/cloudquery/Build?style=flat-square)
```
Comment (dj-stormtrooper) [inline README.md:9]: Refers to this https://github.com/cloudquery/cloudquery/blob/main/.github/workflows/test_build.yml#L1

**Review by bbernays — APPROVED** [review]
LGTM

**Comment by yevgenypats** [comment]
Let's remove it altogether

**Comment by dj-stormtrooper** [comment]
> Let's remove it altogether

Both badges? Licence badge is so sad when it's only one

image

**Comment by erezrokah** [comment]
GitHub already shows the license so I don't think we need the badge:
image

**Review by erezrokah — APPROVED** [review]

**Pull request merged**